content/en/docs/ops/deployment/multicluster-considerations/index.md

```diff
@@ -0,0 +1,179 @@
+without exposing the control plane on the public internet. Such an option, if it exists and meets your requirements,
+will likely be the best choice.
+
+{{< image width="80%"
+    link="remote-clusters.svg"
+    caption="Remote clusters"
+    >}}
+
+## DNS and Service Deployment
+
+When a client application makes a request to some host, it must first perform a DNS lookup for the hostname to
+obtain an IP address before it can proceed with the request.
+
+In Kubernetes, the DNS server residing within the cluster typically handles this DNS lookup. Kubernetes
+services configure this server.
+
+Istio ignores the IP address returned from the DNS lookup, however. Instead, it
+uses the list of active endpoints for the host and load balances across them. Istio uses either Kubernetes
+`Service`/`Endpoint` or Istio `ServiceEntry` to configure its internal mapping of hostname to IP addresses.
+
+This two-tiered naming system becomes more complicated when you have multiple clusters. Istio is inherently
+multicluster-aware, but Kubernetes is not (today). Because of this, for a client in one cluster to make a request to
+a service in another cluster, there must be a Kubernetes service for the host defined in the client cluster in order
+for the DNS lookup to succeed. This is true even if there are no instances of that service's Pods running in the
+client cluster.
+
+To generalize this, you must deploy a Kubernetes service to each cluster that consumes that service. This ensures that
+regardless of where the request originates, it will pass DNS lookup and be handed to Istio for proper routing.
+
+## Networks
+
+Istio uses a simplified definition of [network](/docs/ops/deployment/deployment-models#network-models) to refer to
+pods that have direct reachability. For example, by default all the pods in a single cluster are on a single network.
+
+In multicluster configurations it is not uncommon to have clusters residing in different networks, meaning pods
+in one cluster are not directly reachable from pods in another.
+
+In a multi-network scenario, each cluster will have to expose any services needed by workloads in other networks.
+Cloud vendors may provide options that will not require exposing services on the public internet. Such an option,
+if it exists and meets your requirements, will likely be the best choice.
+
+{{< tip >}}
+Istio currently only supports cross-network communication to workloads with an Istio proxy. This is due to the
+fact that we expose services at Ingress Gateway with TLS pass-through, which enables mTLS directly to the workload.
+A workload without an Istio proxy, however, will likely not be able to participate in mutual authentication with
+other workloads. For this reason, Istio filters out-of-network endpoints for proxyless services.
+{{< /tip >}}
+
+{{< image width="80%"
+    link="east-west.svg"
+    caption="Multiple networks with services exposed through gateway"
+    >}}
+
+### Understanding `MeshNetworks`
```

Comment:
This seems different than the rest of this page. I suggest moving it to the multicluster installation guide, since it is more practical than conceptual, and isn't really a consideration. Also it is cruft from us not making this as easy as we should and will go away :)